pkg/util/arrowtest/arrowtest.go

```diff
@@ -90,6 +88,18 @@ func (rows Rows) Schema() *arrow.Schema {
 	return arrow.NewSchema(fields, nil)
 }
 
+func determineDatatype(value any) arrow.DataType {
+	switch value := value.(type) {
+	case time.Time:
+		return &arrow.TimestampType{Unit: arrow.Nanosecond, TimeZone: value.Location().String()}
+	case *time.Time:
+		return &arrow.TimestampType{Unit: arrow.Nanosecond, TimeZone: value.Location().String()}
```

Comment:
Where do we use `*time.Time`?

Reply:
Never, I just added this in case someone does manage to pass a pointer to a time.Time. But in this case, I don't think that's likely, so I'll take it out.